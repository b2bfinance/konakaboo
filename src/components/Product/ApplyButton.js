import styled from 'styled-components';
import Fab from '@material-ui/core/Fab';

export default styled(Fab)`
  background-color: ${props => props.theme.applyButtonBackground} !important;
  color: ${props => props.theme.applyButtonColor};
  width: 100% !important;
  box-shadow: none !important;

  &:hover {
    color: ${props => props.theme.applyButtonColor};
  }
`;
