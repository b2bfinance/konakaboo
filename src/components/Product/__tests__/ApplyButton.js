import React from 'react';
import { render } from 'enzyme';
import ApplyButton from '../ApplyButton';

describe('ApplyButton', () => {
  test('renders correctly', () => {
    expect(render(<ApplyButton>apply</ApplyButton>)).toMatchSnapshot();
  });
});
