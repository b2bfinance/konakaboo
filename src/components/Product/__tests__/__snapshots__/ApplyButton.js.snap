// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplyButton renders correctly 1`] = `
.c0 {
  background-color: #4caf50 !important;
  color: #fff !important;
  width: 100% !important;
  box-shadow: none !important;
}

.c0:hover {
  color: #fff !important;
}

<button
  class="MuiButtonBase-root-11 MuiFab-root-1 c0"
  tabindex="0"
  theme="[object Object]"
  type="button"
>
  <span
    class="MuiFab-label-2"
  >
    apply
  </span>
</button>
`;
